packages/git/indexer.js

```diff
@@ -107,25 +114,27 @@ module.exports = class Indexer {
 
   async _ensureRepo() {
     if (!this.repo) {
+      if (this.remote) {
+        let tempRepoPath = await mkdir('cardstack-temp-repo');
+        this.repo = await Clone(this.remote.url, tempRepoPath, {
```

Comment:
If I understand correctly, the indexer always clones and never fetches. I don't think that can be correct.

If somebody makes changes to the remote repo (manually, bypassing the hub), and indexing runs, their changes need to get indexed.